### PR TITLE
C376937: Verify "Sets" configuration option is hidden in "OAI-PMH" pane of "OAI-PMH" settings (firebird) (TaaS)

### PR DESCRIPTION
**Pull request by nodosi**
[C376937](https://foliotest.testrail.io/index.php?/cases/view/376937)

[Jenkins CI](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/6220/)

[Allure](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/6220/allure/)

Screenshot 2023-12-21 201000

### CODE REVIEW

File: cypress/e2e/oai-pmh/oai-pmh-pane-sets-config-option-hidden.cy.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  after('Delete test data', () => {
+    cy.getAdminToken();
+    Users.deleteViaApi(testData.user.userId);
+  });
+
+  it(
+    'C376937 Verify "Sets" configuration option is hidden in "OAI-PMH" pane of "OAI-PMH" settings (firebird) (TaaS)',
+    { tags: ['extendedPath', 'firebird'] },
+    () => {
+      TopMenuNavigation.navigateToApp('Settings');
```
Comment (nayimovag) [inline cypress/e2e/oai-pmh/oai-pmh-pane-sets-config-option-hidden.cy.js:30]: this line is redundant, next one also navigates to Settings and then to OAI-PMH

File: cypress/e2e/oai-pmh/oai-pmh-pane-sets-config-option-hidden.cy.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      OaiPmh.checkPageTitle('OAI-PMH settings - FOLIO');
+
+      OaiPmh.selectSection(SECTIONS.GENERAL);
+      OaiPmh.checkPageTitle('OAI-PMH settings - General - FOLIO');
+
+      OaiPmh.selectSection(SECTIONS.TECHNICAL);
+      OaiPmh.checkPageTitle('OAI-PMH settings - Technical - FOLIO');
+
+      OaiPmh.selectSection(SECTIONS.BEHAVIOR);
+      OaiPmh.checkPageTitle('OAI-PMH settings - Behavior - FOLIO');
```
Comment (nayimovag) [inline cypress/e2e/oai-pmh/oai-pmh-pane-sets-config-option-hidden.cy.js:42]: replace lines 33 to 42 with OaiPmh.checkSectionListItems();

File: cypress/e2e/oai-pmh/oai-pmh-pane-sets-config-option-hidden.cy.js (hunk 1 above)
Comment (nodosi) [reply cypress/e2e/oai-pmh/oai-pmh-pane-sets-config-option-hidden.cy.js]: I have removed this line.

File: cypress/e2e/oai-pmh/oai-pmh-pane-sets-config-option-hidden.cy.js (hunk 2 above)
Comment (nodosi) [reply cypress/e2e/oai-pmh/oai-pmh-pane-sets-config-option-hidden.cy.js]: I have replaced lines with this method.

**Review by nayimovag — APPROVED** [review]

**Review by ernestyedigaryan1 — APPROVED** [review]

**Pull request merged**